official/recommendation/ncf_keras_main.py

```diff
@@ -74,8 +75,12 @@ def metric_fn(y_true, y_pred):
             params["match_mlperf"],
             params["use_xla_for_gpu"]))
 
+    is_learning_phase = tf.keras.backend.learning_phase()
```

Comment:
this should be called `is_training` or something. learning_phase is something that can be 1 or 0. 

Why was this change needed now btw?

Reply:
This is not absolutely necessary. It's just that this metric is not used in the training phase, so I set it to zero.